src/azure-cli/azure/cli/command_modules/acs/commands.py

```diff
@@ -71,13 +79,17 @@ def load_command_table(self, _):
                          client_factory=None)
 
     # ACS Kubernetes commands
-    with self.command_group('acs kubernetes', container_services_sdk, client_factory=cf_container_services) as g:
+    with self.command_group('acs kubernetes', container_services_sdk,
+                            resource_type=ResourceType.MGMT_CONTAINERSERVICE,
+                            client_factory=cf_container_services) as g:
```

Comment:
resource_type and client_factory are already defined in container_services_sdk.
```suggestion
    with self.command_group('acs kubernetes', container_services_sdk) as g:
```